.vsts/job.yml

```diff
@@ -105,13 +105,23 @@ jobs:
         publishRunAttachments: true
 
   - ${{ if eq(parameters.name, 'Windows') }}:
+    - task: CmdLine@2
+      condition: always()
```

Comment:
Unnecessary conditions?

Reply:
I didn't realize that was an optional parameter. Thanks :)